src/apps/common/hiredis_libevent2.c

```diff
@@ -233,7 +233,7 @@ redis_context_handle redisLibeventAttach(struct event_base *base, char *ip0, int
   if(ip0 && ip0[0])
 	  STRCPY(ip,ip0);
   else
-	  STRCPY(ip,"127.0.0.1");
+	  strncpy(ip,"127.0.0.1",sizeof(ip));
```

Comment:
"127.0.0.1" is null-terminated so strncpy will copy the null
Longest IP representation is 45 https://stackoverflow.com/a/7477384